specification/src/main/asciidoc/platform/BrandMapping.adoc

```diff
@@ -0,0 +1,114 @@
+[cols=2, options=header]
+.Java EE to Jakarta EE Mapping
+|===
+|Java EE Technology
+|Jakarta EE Technology
+
```

Comment:
These should be 

      Java EE 8 Specification
      Jakarta EE 8 Specification

Reply:
I like Specification better than Technology.

Reply:
Thanks.  Made that change.  I like it too.